.github/workflows/application-signals-e2e-test.yml

```
@@ -180,3 +180,55 @@ jobs:
     with:
       aws-region: us-east-1
       caller-workflow-name: 'main-build'
+
+  dotnet-ec2-default-v8-e2e-test:
+    needs: [ CheckBuildTestArtifacts ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/dotnet-ec2-default-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      caller-workflow-name: 'main-build'
+      dotnet-version: '8.0'
+
+  dotnet-ec2-default-v6-e2e-test:
+    needs: [ CheckBuildTestArtifacts ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/dotnet-ec2-default-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      caller-workflow-name: 'main-build'
+      dotnet-version: '6.0'
+
+  dotnet-ec2-windows-e2e-test:
+    needs: [ CheckBuildTestArtifacts ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/dotnet-ec2-windows-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      caller-workflow-name: 'main-build'
+
+  dotnet-eks-e2e-test:
+    needs: [ CheckBuildTestArtifacts ]
```

All the tests share one EKS cluster, make sure this runs after the previous eks test in the list
```suggestion
    needs: [ CheckBuildTestArtifacts, node-eks-e2e-test ]
```

---

I have made this change as well